components/notebook-controller/Makefile

```diff
@@ -61,7 +61,7 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./api/...
 
 # Build the docker image
-docker-build: test
+docker-build:
```

Comment:
if it's just the `check-license` portion that's failing, perhaps we can just omit that until we [update the licenses](https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/go-license-tools/README.md)

Reply:
yes it was the check-license part that’s failing